### PR TITLE
Jesse/issue 2

### PR DESCRIPTION
**Pull request by vc-ca**
httpOnly cookies are working properly. I did not complete CSRF in order to allocate enough time for Issue #3.

On server side: 
Anytime the JWT is issued, app now returns a token cookie.
CookieParser package was installed to read cookie tokens, rather than the X-access auth header tokens.
Server now reads tokens not from Auth header, but req.body.token.

Client side:
Removed axios headers & local storage.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -18,9 +20,11 @@ app.use(logger("dev"));
 app.use(json());
 app.use(urlencoded({ extended: false }));
 app.use(express.static(join(__dirname, "public")));
+app.use(cookieParser());
 
 app.use(function (req, res, next) {
-  const token = req.headers["x-access-token"];
+  //replaced the X-Access header token with the cookie parser token available on req body
```
Comment (RayK3) [inline server/app.js:26]: It's good to include high level comments, but here and your README both say that the token is coming from the `req.body` when it's actually being set by the `cookieParser` middleware and coming from `req.cookies`.

File: server/routes/auth/index.js [hunk 2]
```diff
@@ -26,6 +26,8 @@ router.post("/register", async (req, res, next) => {
       process.env.SESSION_SECRET,
       { expiresIn: 86400 }
     );
+    res.cookie("token", token, { httpOnly: true });
```
Comment (RayK3) [inline server/routes/auth/index.js:29]: These cookies should be given the same expiration time as the token 👍

**Review by RayK3 — COMMENTED** [review]
Looks good, this is a good way to use cookies. I only had a couple of comments 👍

File: server/app.js (hunk 1 above)
Comment (vc-ca) [reply server/app.js:26]: Noted!

File: server/routes/auth/index.js (hunk 2 above)
Comment (vc-ca) [reply server/routes/auth/index.js:29]: Thanks, I've updated the expiration in the PR

**Review by bonnieli — APPROVED** [review]

**Pull request merged**